apps/web/components/apps/App.tsx

```diff
@@ -24,6 +25,9 @@ import {
   FiShield,
 } from "@calcom/ui/components/icon";
 
+/* These app slugs all require Google Cal to be installed */
+const appsThatRequiresGCal = ["google-meet", "amie", "vimcal"];
```

Comment:
can we somehow move this into app specific configs?

ideally i dont want to hard code app things into core components 

@hariombalhara any idea?

Reply:
Perhaps adding a property to the app's metadata?

Reply:
yep, easy


we may in the future have other apps that require other apps

Reply:
We have a similar situation right now which is Typeform requiring Routing Forms, but in that case we didn't need such a config as Typeform App's UI itself exists in Routing Form App's UI.

@joeauyeung  Yeah, that should be the right place.